components/tabs/tabs.js

```diff
@@ -393,11 +397,30 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 		return this.shadowRoot.querySelector('.d2l-tabs-container-list').getBoundingClientRect();
 	}
 
+	hideTab(tab) {
+		tab.setAttribute('data-state', 'removing');
+		return (Object.keys(this._tabIds).length > 1 && !reduceMotion) ? this._animateTabRemoval(tab) : Promise.resolve();
+	}
+
 	#checkTabPanelMatchRequested;
 	#panels;
 	#updateAriaControlsRequested;
 
-	_animateTabAddition(tabInfo) {
+	_animateTabAddition(tab) {
+		return new Promise((resolve) => {
```

Comment:
Looks like these animations honour prefers reduced motion. If the user prefers reduced motion, then I think the transition events won't happen, the `Promise` never resolves, and the event never gets un-wired. Sorta [like this](https://github.com/BrightspaceUI/core/blob/main/mixins/visible-on-ancestor/visible-on-ancestor-mixin.js#L110), but I think there a re bunch of examples in core.

Reply:
Yup so we don't actually add the `adding` state to them [if the user prefers reduced motion](https://github.com/BrightspaceUI/core/pull/5471/files/465cc137c8d95f1a7e2bcd292a4fe1baf3ae2e19#diff-ce8cc2dbbd772bf9580235f279774d67c5a1c1f2ecb6f1b7afe0f8d3e67eef70R820), so they never run the animation

Reply:
Oh ok, perfect. Thanks.